commands/team_users_e2e_test.go

```diff
@@ -159,3 +159,54 @@ func (s *MmctlE2ETestSuite) TestTeamUserAddCmd() {
 		s.Require().Equal(printer.GetErrorLines()[0], fmt.Sprintf("Can't find user '%s'", nonexistentUserEmail))
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestTeamUsersRemoveCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	user, appErr := s.th.App.CreateUser(&model.User{Email: s.th.GenerateTestEmail(), Username: model.NewId(), Password: model.NewId()})
+	s.Require().Nil(appErr)
+
+	team := model.Team{
+		DisplayName: "dn_" + model.NewId(),
+		Name:        api4.GenerateTestTeamName(),
+		Email:       s.th.GenerateTestEmail(),
+		Type:        model.TEAM_OPEN,
+	}
+	_, appErr = s.th.App.CreateTeamWithUser(&team, user.Id)
+	s.Require().Nil(appErr)
+
+	s.RunForSystemAdminAndLocal("Remove user from team", func(c client.Client) {
+		printer.Clean()
+
+		err := teamUsersRemoveCmdF(c, &cobra.Command{}, []string{team.Name, user.Username})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+
+		teamMembers, err := s.th.App.GetTeamMembers(team.Id, 0, 10, nil)
+		s.Require().Nil(err)
+		s.Require().NotNil(teamMembers)
+		s.Require().Len(teamMembers, 0)
```

Comment:
You'll need to move the user creation and addition to team inside the test. `RunForSystemAdminAndLocal` runs the test function twice. First as SystemAdmin and second time as Local Client.

The SystemAdmin test run removes the user successfully. For the local client test run the user has already been removed from the team so it always passes. mmctl doesn't complain if a user doesn't already exist.

Reply:
Thanks @ashishbhate, good catch! I moved it and committed in a `fixup` which I can rebase if/when you're 👍 on it.